website/docs/r/repository.html.markdown

```diff
@@ -63,6 +63,8 @@ The following arguments are supported:
 * `has_issues` - (Optional) Set to `true` to enable the GitHub Issues features
   on the repository.
 
+* `has_discussions` - (Optional) Set to `true` to enable GitHub Discussions on the repository.
```

Comment:
Thank you for contributing! Would you mind editing this to give the default value?

Reply:
@nbaechtold, I went ahead and got that change that @kfcampbell requested.  Thank you for this change! ❤️